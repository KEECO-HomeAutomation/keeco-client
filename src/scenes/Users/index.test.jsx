import React from 'react';
import { shallow } from 'enzyme';

import Users from './index';

describe('<Users />', () => {
	test('Should render without errors', () => {
		const comp = shallow(<Users />);
		expect(comp.length).toBe(1);
	});

	test('Should match snapshot', () => {
		expect(shallow(<Users />)).toMatchSnapshot();
	});
});
