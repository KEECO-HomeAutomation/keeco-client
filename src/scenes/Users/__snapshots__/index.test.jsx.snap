// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Users /> Should match snapshot 1`] = `
<p>
  Users
</p>
`;
